Automattic/wp-calypso
# Posts: Add single site post requesting behavior to Redux state

This pull request seeks to implement Redux state behaviors for managing network requests of single site post objects. This will be used as part of the editor Redux migration effort in allowing the editor to reconcile post revisions with the persisted post entity.

__Testing instructions:__

There are no effective visual changes to test.

Ensure Mocha tests pass by running `make test` in your Terminal from the project root directory or directly from `client/state/posts`.
